### DISCUSSION TITLE
Custom module mapping to accounts client name

### DISCUSSION THREAD

**Discussion by MonsterITServices** [YetiForceCompany/YetiForceCRM#16711] [How to]
Hi, We have a custom module added but cannot seem to map the accounts client name to the client name in the custom module. I've tried looking for the documentation on how to do this but it appears missing/not there.  How can I do this please? I'm running version 6.4 on ubuntu apache litespeed server.

**Comment by apcloic** [comment]
@alextovey 
Hi,
If I understand correctly what you want to do, I think you only have to create a 1:M or M:M relation field in your custom module and it should work.

![account_relation_custom_module](https://user-images.githubusercontent.com/2930560/205949383-3ef51a74-4c30-4b28-994d-32c97eacde16.gif)

Regards,
